services-html.md

```diff
@@ -67,8 +67,8 @@ The second argument of `Form::ajax` should contain the attributes:
 You can also pass partials to update as another array:
 
     Form::ajax('onSave', ['update' => [
-            'control-panel': '#controlPanel',
-            'layout/sidebar': '#layoutSidebar'
+            'control-panel' => '#controlPanel',
+            'layout/sidebar' => '#layoutSidebar'
         ]
     ])
```

Comment:
Could you put a semicolon at the end of the statement as well to cover it off perfectly then?

Reply:
Looks alright I think, nothing else in the doc has semi colons.